### PR TITLE
Ci check unused again

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: ci/build-check.sh [hunk 1]
```diff
@@ -15,6 +15,9 @@ if test -x /usr/bin/gnome-desktop-testing-runner; then
 fi
 
 if test -x /usr/bin/clang; then
+    # always fail on warnings; https://github.com/ostreedev/ostree/pull/971
+    # Except for clang-4.0: error: argument unused during compilation: '-specs=/usr/lib/rpm/redhat/redhat-hardened-cc1' [-Werror,-Wunused-command-line-argument]
+    export CFLAGS="Wno-error=unused-command-line-argument -Werror ${CFLAGS:-}"
```
Comment (jlebon) [inline ci/build-check.sh]: I think this may be causing `configure` to barf. Let's artifact `config.log` as well for a closer look?

Comment (cgwalters) [inline ci/build-check.sh]: Yeah, dunno what's going on, WFM locally.

File: ci/build-check.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ fi
 if test -x /usr/bin/clang; then
     # always fail on warnings; https://github.com/ostreedev/ostree/pull/971
     # Except for clang-4.0: error: argument unused during compilation: '-specs=/usr/lib/rpm/redhat/redhat-hardened-cc1' [-Werror,-Wunused-command-line-argument]
-    export CFLAGS="Wno-error=unused-command-line-argument -Werror ${CFLAGS:-}"
+    export CFLAGS="-Wno-error=unused-command-line-argument -Werror ${CFLAGS:-}"
```
Comment (jlebon) [inline ci/build-check.sh]: Haha, can't believe that was it.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ ea225b8

**Pull request closed without merging**